src/backend.rs

```diff
@@ -120,6 +125,31 @@ impl BackendDetector {
     }
 }
 
+/// Check if policy server version is compatible with  minimum kubewarden
+/// version required by the policy
+pub fn has_minimum_kubewarden_version(metadata: &Metadata) -> Result<()> {
+    if let Some(minimum_kubewarden_version) = &metadata.minimum_kubewarden_version {
+        let sanitized_minimum_kubewarden_version = Version {
+            major: minimum_kubewarden_version.major,
+            minor: minimum_kubewarden_version.minor,
+            // Kubewarden stack version ignore patch version number
+            patch: 0,
+            // prerelease and build objects does not implement copy trait.
+            // Thus, we need to clone them
+            pre: minimum_kubewarden_version.pre.clone(),
+            build: minimum_kubewarden_version.build.clone(),
```

Comment:
This has to be fixed like you did with policy-server. Using `empty`. Make sure the comment is dropped

Reply:
Ops... I forgot to change that here. Fixing now...